File: extension/gsa/fixtures/web/hard/web-hard-01-api-access-boundary/workspace/src/caseContext.ts
export interface CaseContext {
  id: string;
  category: string;
  difficulty: string;
  reviewStream: string;
  localOnly: boolean;
}

export const caseContext: CaseContext = {
  "id": "web-hard-01-api-access-boundary",
  "category": "web",
  "difficulty": "hard",
  "scenario": "api-access-boundary",
  "reviewStream": "stream-web-hard-1",
  "localOnly": true
};

export function caseLabel(): string {
  return caseContext.category + ':' + caseContext.difficulty + ':' + caseContext.id;
}

File: extension/gsa/fixtures/network-security/hard/network-security-hard-01-service-exposure-review/workspace/src/network_security_service_exposure_review/case_context.py
from dataclasses import dataclass

@dataclass(frozen=True)
class CaseContext:
    id: str
    category: str
    difficulty: str
    scenario: str
    review_stream: str
    local_only: bool

CONTEXT = CaseContext(
    id="network-security-hard-01-service-exposure-review",
    category="network-security",
    difficulty="hard",
    scenario="service-exposure-review",
    review_stream="stream-network-security-hard-1",
    local_only=True,
)

def label() -> str:
    return f"{CONTEXT.category}:{CONTEXT.difficulty}:{CONTEXT.id}"
